### DISCUSSION TITLE
ios jumping footer and white bar at the bottom

### DISCUSSION THREAD

**Discussion by JumaVandam** [quasarframework/quasar#10481] [CLI - Capacitor / Cordova modes] [answered]
<img width="354" alt="скачущий футэр" src="https://user-images.githubusercontent.com/82858978/130923834-7d1eed22-4e85-4c92-a7d7-da1690c79828.png">
Hello everyone, I put together a build on ios and faced the problem of a jumping footer on some pages, and when I open the left menu, a white bar appears at the bottom and I can't find a solution to this problem, please help.

**Comment by pdanpdan** [comment, marked as answer]
What version of quasar, 1 or 2?

**Reply by JumaVandam** [reply]
version quasar is 1. I just now accidentally solved the problem by adding it to app.css this: 
.q-body--prevent-scroll.platform-ios {
	overflow: hidden;
        height: 1px;
	position: relative !important;
}
i dont understand how but it working.
I find this solution here: [https://github.com/quasarframework/quasar/issues/6695](url)

**Comment by pdanpdan** [comment]
If you can do a test for me (only for quasar v1)

https://github.com/pdanpdan/quasar/releases/tag/quasar-pdan-v1.15.23-beta.6

In package.json replace (or add) quasar dependency with:
"quasar": "https://github.com/pdanpdan/quasar#quasar-pdan-v1.15.23-beta.6"

Then do a yarn or npm install and start the app again.

Please report if you find something not working.
